app/models/tiny_mce_asset.rb

```diff
@@ -170,7 +170,10 @@ def clone_tinymce_asset(obj)
 
     tiny_img_clone.transaction do
       tiny_img_clone.save!
-      tiny_img_clone.image.attach(io: image.download, filename: image.filename.sanitized)
+      tiny_img_clone.image.attach(io: StringIO.new(image.download),
```

Comment:
I would use temp files instead of loading file content to memory as file size is limited by FILE_MAX_SIZE_MB and it can be quite big